core/trino-main/src/main/java/io/trino/sql/planner/planprinter/JsonRenderer.java

```diff
@@ -14,11 +14,14 @@
 package io.trino.sql.planner.planprinter;
```

Comment:
Commit message is wrong. It's no longer an identifier but descriptor

Reply:
Fixed the commit message